platform/ios/src/MGLMapView.mm

```diff
@@ -800,9 +801,20 @@ - (void)setScaleBarPosition:(MGLOrnamentPosition)scaleBarPosition {
 - (void)setScaleBarMargins:(CGPoint)scaleBarMargins {
     MGLLogDebug(@"Setting scaleBarMargins: (x:%f, y:%f)", scaleBarMargins.x, scaleBarMargins.y);
     _scaleBarMargins = scaleBarMargins;
+    if([self.scaleBar isKindOfClass:[MGLScaleBar class]]) {
+        ((MGLScaleBar *)self.scaleBar).margins = scaleBarMargins;
+    }
     [self installScaleBarConstraints];
 }
 
+- (void)setScaleBarMaximumWidthRatio:(CGFloat)scaleBarMaximumWidthRatio {
```

Comment:
Can this test be included within `testChangeScaleBarSize`?

Reply:
Correct me if I'm wrong but I'm not sure if the tests cover the case where after this method is called, the scale bar adjusts accordingly.